reference/component-versions.md

```diff
@@ -14,53 +14,53 @@ Not all components are updated with each release. When components are not update
+| `coturn` | `1.28/stable` | 1087 | 1088 | `1.28.0-0fa313f` |
+| `ams` | `1.28/stable` | 1282 | 1283 | `1.28.0-0fa313f` |
+| `anbox-stream-gateway` | `1.28/stable` | 1206 | 1207 | `1.28.0-0fa313f` |
+| `ams-lxd` | `1.28/stable` | 1170 | 1171 | `1.28.0-0fa313f` |
+| `lxd-integrator` | `1.28/stable` | 565 | 566 | `1.28.0-0fa313f` |
+| `anbox-stream-agent` | `1.28/stable` | 1199 | 1200 | `1.28.0-0fa313f` |
+| `aar` | `1.28/stable` | 1270 | 1271 | `1.28.0-0fa313f` |
+| `anbox-cloud-cos-configuration` | `1.28/stable` | 397 | 398 | `1.28.0-0fa313f` |
 
 
 #### Ubuntu 24.04
 
 | Name | Channel | Revision (AMD64) | Revision (ARM64) | Version |
 |----------|--------------|--------------|-------------|---------|
-| `anbox-cloud-dashboard` | `1.28/stable` |  |  |  |
-| `coturn` | `1.28/stable` |  |  |  |
-| `ams` | `1.28/stable` |  |  |  |
-| `anbox-stream-gateway` | `1.28/stable` |  |  |  |
-| `ams-lxd` | `1.28/stable` |  |  |  |
-| `lxd-integrator` | `1.28/stable` |  |  |  |
-| `anbox-stream-agent` | `1.28/stable` |  |  |  |
-| `aar` | `1.28/stable` |  |  |  |
-| `anbox-cloud-cos-configuration` | `1.28/stable` |  |  |  |
+| `anbox-cloud-dashboard` | `1.28/stable` | 1116 | 1117 | `1.28.0-0fa313f` |
+| `coturn` | `1.28/stable` | 1089 | 1090 | `1.28.0-0fa313f` |
+| `ams` | `1.28/stable` | 1284 | 1285 | `1.28.0-0fa313f` |
+| `anbox-stream-gateway` | `1.28/stable` | 1208 | 1209 | `1.28.0-0fa313f` |
+| `ams-lxd` | `1.28/stable` | 1172 | 1173 | `1.28.0-0fa313f` |
+| `lxd-integrator` | `1.28/stable` | 567 | 568 | `1.28.0-0fa313f` |
+| `anbox-stream-agent` | `1.28/stable` | 1201 | 1202 | `1.28.0-0fa313f` |
+| `aar` | `1.28/stable` | 1272 | 1273 | `1.28.0-0fa313f` |
+| `anbox-cloud-cos-configuration` | `1.28/stable` | 399 | 400 | `1.28.0-0fa313f` |
 
 
 ### Bundles
 
 | Name | Channel | Revision |
 |----------|--------------|--------------|
-| `anbox-cloud` | `1.28/stable` |  |
-| `anbox-cloud-core` | `1.28/stable` |  |
+| `anbox-cloud` | `1.28/stable` | 884 |
+| `anbox-cloud-core` | `1.28/stable` | 897 |
 
 
 ### Snaps
 
 | Name | Channel | Revision (AMD64) | Revision (ARM64) | Version |
 |------|---------|------------------|------------------|---------|
-| `ams` | 1.28/stable |  |  | `1.28.0-504d447de` |
-| `ams-node-controller` | 1.28/stable |  |  | `1.28.0-e373298aa` |
-| `amc` | latest/stable |  |  | `1.28.0-504d447de` |
-| `aar` | 1.28/stable |  |  | `1.28.0-504d447de` |
-| `anbox-stream-agent` | 1.28/stable |  |  | `1.28.0-504d447de` |
-| `anbox-stream-gateway` | 1.28/stable |  |  | `1.28.0-504d447de` |
-| `anbox-cloud-appliance` | 1.28/stable |  |  | `1.28.0-504d447de` |
-| `anbox-cloud-dashboard` | 1.28/stable |  |  | `1.28.0-504d447de` |
-| `anbox-connect` | latest/stable |  |  | `1.28.0-504d447de` |
+| `ams` | `1.28/candidate` | 2562 | 2563 | `1.28.0-504d447de` |
+| `ams-node-controller` | `1.28/candidate` | 2139 | 2140 | `1.28.0-504d447de` |
+| `amc` | `latest/candidate` | 2358 | 2359 | `1.28.0-504d447de` |
```

Comment:
Just wondering why those snaps have to come from the `<track>/candidate` channel?